src/lib.rs

```diff
@@ -168,8 +168,12 @@ where
         Ok(())
     }
 
+    /// Sets pixel colors in any given rectangle bounds. Does not perform any bounds checking on the input color iterator.
+    /// It's not meat to be used directly, it's exposed for frameworks to be able to use it.
```

Comment:
The first line of doc comments should be a summary (https://doc.rust-lang.org/1.0.0/style/style/comments.html#summary-line). 

Here is another attempt to make this easier to understand:
```suggestion
    /// Sets pixel colors in a rectangular region.
    ///
    /// The color values from the `colors` iterator will be drawn to the given region starting
    /// at the top left corner and continuing, row first, to the bottom right corner. No bounds
    /// checking is performed on the `colors` iterator and drawing will wrap around if the
    /// iterator returns more color values than the number of pixels in the given region.
    ///
    /// This is a low level function, which isn't intended to be used in regular user code.
    /// Consider to use the drawing functions provided by the `embedded-graphics` crate instead.
```
GitHub didn't allow me to add a suggestion which includes all lines until the "Arguments" heading, but this is meant to replace the entire first part of the comment (up to the "Arguments" heading).

I did leave out the detail about using `fill_contiguous`, because that is also a low level function which shouldn't be used directly.